files/paths.md

```diff
@@ -0,0 +1,25 @@
+## Path Definition Information
+### A path is a collection of routes that serve a common corridor or origin-destination pair
+
+ *  Filename MUST be `paths.txt`
+ *  File MUST contain a record for every transit route and in a path
+ *  File MUST be a valid CSV file.
+ *  The first line of each file MUST contain case-sensitive field names.
+ *  Field names MUST NOT contain tabs, carriage returns or new lines.
+
+File MUST contain the following required attributes:
+
+Required Attributes	| Description										
+----------			| -------------		
+`path_id`			| ID that uniquely identifies a path.
+`route_id`			| ID that uniquely identifies a route.
```

Comment:
If we want to accumulate among many routes, would we have to denote every route that is in the group? Just trying to think if there is a way to do this automagically (i.e. by selecting from a set of nodes that it must traverse)

Reply:
Yes.  If you think about Market Street there are curb-side and island stops with different stop ids but that you might want to include in a single corridor, which would be difficult to do if you used stop sequences.  I think there's probably a trade off between flexibility and automation.  I'm open to reconsidering

Reply:
what about creating lists where everything across the line is an AND but within the list is an OR?
set_id, from_node_set, to_node_set, thru_node_set
castro_VN, [castro_muni], [van_ness_muni],[]
castro_VN,[castro/17th, 17th/castro etc.],[van_ness/Market, Market/Van Ness, etc], []

Reply:
What if you wanted to make sure somethings was NOT included?  For example, maybe you want `Geary_ParkPresidio_Market` but want to exclude the 91?

Reply:
oh nevermind...